copter/source/docs/autotrim.rst

```diff
@@ -25,54 +25,49 @@ Save trim is the simpler method and involves essentially transferring your radio
-
-2. Put the vehicle in Stabilize mode
-
-3. Hold throttle down and rudder right for 15seconds or until you see the small red, blue and yellow leds flash in a cyclic pattern
+.. note::
 
-4. Fly your copter for about 25 seconds in a stable hover
+     You can manually set the trim by modifying the :ref:`AHRS_TRIM_X <AHRS_TRIM_X>` and :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Roll trim is :ref:`AHRS_TRIM_X <AHRS_TRIM_X>`, Pitch trim is :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Both values are in radians with left roll and forward pitch being negative numbers.
 
-5. Land and put your throttle to zero and wait for a few seconds (the trims parameters are being saved)
+.. note::
 
-6. Take off again in stabilize mode and check if your copter is flying level now. If not repeat steps 2, 3 and 4
+    It is nearly impossible to get rid of all drift so that your copter remains completely motionless without any input.
 
-.. note::
+Save Trim
+~~~~~~~~~
+.. note:: the above AUTO TRIM method is preferred since it does not involve moving transmitter trims after RC calibration.
 
-    You can also test these procedures above are operational on the ground with your battery disconnected. Connect your autopilot to the mission planner and observing the Flight Data screen as you simulate completing the steps above.
+Save trim involves essentially transferring your radio transmitter's trims to correct the autopilot's accelerometer trim. Normally, once :ref:`common-accelerometer-calibration` is done, you never want to change the transmitter's trims, but in this case it is done **temporarily**. The "Save Trim" function ("5") is assigned to any free RC channel, usually a switch based one, using an :ref:`common-auxiliary-functions`. For the example below, Channel 7 will be used.
 
-.. image:: ../images/MP_SaveTrim_FlightDataScreen.jpg
-    :target: ../_images/MP_SaveTrim_FlightDataScreen.jpg
+1. Check that your CH7 switch goes above 1800 on the MissionPlanner's Hardware > Mandatory Hardware > Radio Calibration screen
 
-.. note::
+.. image:: ../images/MP_SaveTrim_Ch7PWMCheck.png
+    :target: ../_images/MP_SaveTrim_Ch7PWMCheck.png
 
-    You can manually set the trim by modifying the :ref:`AHRS_TRIM_X <AHRS_TRIM_X>` and :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Roll trim is :ref:`AHRS_TRIM_X <AHRS_TRIM_X>`, Pitch trim is :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Both values are in radians with left roll and forward pitch being negative numbers.
+2. Set the CH7 Option to Save Trim on the Software > Copter Pids screen and press the "Write Params" button
 
-.. note::
+.. image:: ../images/MP_SaveTrim_Ch7.png
+    :target: ../_images/MP_SaveTrim_Ch7.png
 
-    It is nearly impossible to get rid of all drift so that your copter remains completely motionless without any input.
+3. With your CH7 switch in the off position, fly your copter in Stabilize mode and use your transmitters's roll and pitch trim to get it flying level
```

Comment:
.. and once more replace "CH7" here.. or I guess at the top of the instructions we could specify to set RC7_OPTION = "5"